Effects/FlxFloodFill/source/FlxFloodFill.hx

```diff
@@ -0,0 +1,131 @@
+/**
+ * 
+ * A HaxeFlixel port of Photonstorm's 'FloodFillFX':
+ * https://github.com/photonstorm/Flixel-Power-Tools/blob/master/src/org/flixel/plugin/photonstorm/FX/FloodFillFX.as
+ * 
+ */
+
+package;
+
+import flash.display.BitmapData;
+import flash.geom.Point;
+import flash.geom.Rectangle;
+import flixel.FlxG;
+import flixel.FlxSprite;
+import flixel.util.FlxColor;
+import flixel.util.FlxDestroyUtil;
+
+/**
+ * "Creates a flood fill effect FlxSprite, useful for bringing in images in cool ways"
+ */
+class FlxFloodFill extends FlxSprite
+{
+	private var complete:Bool = false;
+	private var isFilling:Bool = false;
+	private var dropRect:Rectangle;
+	private var dropPoint:Point;
+	private var dropY:Int;
+	private var srcBitmapData:BitmapData;
+	private var fillDelay:Float = .05;
+	private var fillClock:Float = 0;
+	
+	/**
+	 * How many pixels to drop per update
+	 */
+	private var fillOffset:Int = 1;
+	
+	/**
+	 * @param _x The effect's x-position.
+	 * @param _y The effect's y-position.
+	 * @param _srcBmd The image data used for the effect.
+	 * @param _width The width of the effect.
+	 * @param _height The height of the effect (a value larger than the source image makes the effect taller!).
+	 * @param _fillLinesPerUpdate Number of lines per update to fill the effect with.
+	 * @param _delayPerUpdate The time delay between each update.
+	 */
+	public function new(_x:Float, _y:Float, _srcBmd:BitmapData, ?_width:Int, ?_height:Int, ?_fillLinesPerUpdate:Int = 1, ?_delayPerUpdate:Float = .05)
```

Comment:
To not confuse parameter x/y with the sprites's x/y and the others for consistency. Made them all use "lowerCamelCase" instead.